pom.xml

```diff
@@ -87,6 +87,12 @@
 
     <dependencyManagement>
         <dependencies>
+            <dependency>
```

Comment:
Euhm, don't remember why I added this. We should be getting it as a transitive dependency. But might be a way to resolve conflict between driver and embedded Cassandra. Find out and add comment?

Reply:
Removed this dependency and all tests still pass... And there is no problem running ecAudit in CCM either... So I guess we can remove it.

Reply:
Agree